File: percent.c
#include <stdio.h>
#include <unistd.h>

int main(){
    int n,x,y;
    float m,i;
    
    printf("SEND SUM");scanf("%f", &m);printf("PERCENT ");scanf("%f", &i);printf("YEARS ");scanf("%d", &n);
    i = i/100;
    for (x = 0; n>x; x = x + 1) {m = m+m*i;printf("%d \n",(int)m);}
    sleep(5);
    return 0; 
}

File: percent.pas
program CUBE; 
uses CRT;
var
m:double; //SUM 
i:double; //PERCENT
n:integer; //YEARS 
//CYCLES
x:integer;
y:longint;
begin
writeln('SEND SUM');read(m);writeln('PERCENT ');read(i);writeln('YEARS ');read(n);
i:=(i/100);
for x:=n downto 1 do 
 begin
 m:= m+m*i;writeln(m:0:0);
 end;
writeln('FINAL RESULT ');writeln(m:0:0);
for y:=10 downto 1 do Delay(100);
end.
